docs/configuration.rst

```diff
@@ -1709,6 +1709,10 @@ Example:
     Setting ``needs_reproducible_json = True`` will ensure the JSON output is reproducible,
     e.g. by removing timestamps from the output.
 
+.. versionadded:: 2.1.0
```

Comment:
Shouldn't `needs_json_remove_defaults` get its own headline?

Currently, we have for (nearly) all config options an extra section, so that it shows up in the toc.
This feature can also be used with ``make needs``. So, it's not only related to the containing section of `needs_build_json`

Reply:
done